charms/kubernetes_snaps.py

```diff
@@ -384,6 +384,7 @@ def create_kubeconfig(dest, ca, server, user, token):
 
 def create_service_account_key():
     dest = "/root/cdk/serviceaccount.key"
```

Comment:
i noticed `"/root/cdk/serviceaccount.key"` is used twice in this file.  Perhaps there are other uses. 

What if you pull out

```python
SERVICE_ACCOUNT_KEY = Path("/root/cdk/serviceaccount.key")
```

as a constant?